### PR TITLE
PMM-10119 Update the list of metrics collected

### PR DESCRIPTION
**Pull request by rnovikovP**
to make a list aligned with https://github.com/percona/pmm/blob/main/managed/services/telemetry/config.default.yml

### CODE REVIEW

File: docs/how-to/configure.md [hunk 1]
```diff
@@ -78,11 +78,19 @@ The *Telemetry* switch enables gathering and sending basic **anonymous** data to
...
+- PMM Server Backup feature enabled/disabled
+- PMM Server DBaaS feature enabled/disabled
+- PMM Server Check Updates feature disabled
+- Version of monitored MySQL services
+- Version of monitored MongoDB services
+- Version of monitored PostgreSQL services
+- Grafana total users
+- Monitored nodes count
+- Monitored services count
+- Agents Versions
```
Comment (rasika-chivate) [inline docs/how-to/configure.md:92]:
```suggestion
- Agents version
```

File: docs/how-to/configure.md [hunk 2]
```diff
@@ -78,11 +78,19 @@ The *Telemetry* switch enables gathering and sending basic **anonymous** data to
...
-- Installation Method (Docker, AMI, OVF);
-- the Server Uptime;
-- Advisors (enabled or disabled);
-- Integrated Alerting Status (enabled or disabled).
+- PMM Server Integration Alerting feature enabled/disabled
+- PMM Server Security Thread Tool feature enabled/disabled
+- PMM Server Backup feature enabled/disabled
+- PMM Server DBaaS feature enabled/disabled
+- PMM Server Check Updates feature disabled
+- Version of monitored MySQL services
```
Comment (rasika-chivate) [inline docs/how-to/configure.md:86]:
```suggestion
- Monitored MySQL services version
```

File: docs/how-to/configure.md [hunk 3]
```diff
@@ -78,11 +78,19 @@ The *Telemetry* switch enables gathering and sending basic **anonymous** data to
...
+- Version of monitored MongoDB services
```
Comment (rasika-chivate) [inline docs/how-to/configure.md:87]:
```suggestion
- Monitored MongoDB services version
```

File: docs/how-to/configure.md [hunk 4]
```diff
@@ -78,11 +78,19 @@ The *Telemetry* switch enables gathering and sending basic **anonymous** data to
...
+- Version of monitored PostgreSQL services
```
Comment (rasika-chivate) [inline docs/how-to/configure.md:88]:
```suggestion
- Monitored PostgreSQL services version
```

File: docs/how-to/configure.md [hunk 5]
```diff
@@ -78,11 +78,19 @@ The *Telemetry* switch enables gathering and sending basic **anonymous** data to
...
+- Grafana total users
```
Comment (rasika-chivate) [inline docs/how-to/configure.md:89]:
```suggestion
- Total Grafana users
```

**Review by rasika-chivate — CHANGES_REQUESTED** [review]

File: docs/how-to/configure.md [hunk 6]
```diff
@@ -78,11 +78,19 @@ The *Telemetry* switch enables gathering and sending basic **anonymous** data to
...
-- Installation Method (Docker, AMI, OVF);
-- the Server Uptime;
-- Advisors (enabled or disabled);
-- Integrated Alerting Status (enabled or disabled).
+- PMM Server Integration Alerting feature enabled/disabled
+- PMM Server Security Thread Tool feature enabled/disabled
+- PMM Server Backup feature enabled/disabled
+- PMM Server DBaaS feature enabled/disabled
+- PMM Server Check Updates feature disabled
+- Detailed information about version and comment of monitored MySQL services
```
Comment (rasika-chivate) [inline docs/how-to/configure.md:86]: I am unsure what this means "comment of monitored MySQL services"

Comment (rnovikovP) [reply docs/how-to/configure.md:86]: There are now two metrics version and "version comments" ... this is more extended description of the version, so we can see is this community or Percona version of MySQL

Comment (rasika-chivate) [reply docs/how-to/configure.md:86]: Detailed information should contain everything ideally. " Detailed information about the version of monitored MySQL services"

Comment (rnovikovP) [inline docs/how-to/configure.md:86]:
```suggestion
- Detailed information about the version of monitored MySQL services
```

**Review by rasika-chivate — APPROVED** [review]

**Review by rasika-chivate — APPROVED** [review]

**Pull request merged**